crates/js-sys/tests/wasm/Number.rs

```diff
@@ -1,4 +1,7 @@
-use std::f64::{INFINITY, NAN};
+use std::{
+    convert::TryFrom,
+    f64::{INFINITY, NAN},
```

Comment:
```suggestion
```
I assume this was from an earlier iteration?

Reply:
I'm not sure what you mean here? These imports were already present before my changes, and do seem to be required

Reply:
Apologies, I was sleep walking apparently :sweat_smile:!